app/controllers/charges_controller.rb

```diff
@@ -32,13 +32,18 @@ def create
         sum + item['amount'] * item['quantity']
       end
 
+    is_distribution = !charge_params[:is_distribution].nil? ?
```

Comment:
.present?

Reply:
`is_distribution = charge_params[:is_distribution] || false`?

Reply:
ah that's so much cleaner smart